docs/source/noise/column_noise.rst

```diff
@@ -4,18 +4,145 @@
+or the answer was illegible or unintelligible.
+When this type of noise occurs, pseudopeople will replace the value in the relevant cell with
+:code:`numpy.nan` to indicate that the value is missing.
+
+This noise type is called :code:`leave_blank` in the configuration. It takes one parameter:
+
+.. list-table:: Parameters to the leave_blank noise type
+  :widths: 1 5 1
+  :header-rows: 1
+
+  * - Parameter
+    - Description
+    - Default
+  * - :code:`cell_probability`
+    - The probability that a cell in the column being configured is blank.
+    - 0.01 (1%)
+
 Choose the wrong option
 -----------------------
 
-Write the wrong zipcode digits
-------------------------------
+If a question on a survey or administrative form provides a list of options,
+respondents may sometimes choose the wrong option, either intentionally or by mistake.
+pseudopeople simulates this type of noise by sometimes selecting an incorrect option
+for columns that would have a list of options.
+All wrong options are equally likely.
+The possible values to select from depend on the column:
+the :ref:`Datasets page <datasets_main>` lists them for each applicable column in pseudopeople's datasets.
+
+This noise type is called :code:`choose_wrong_option` in the configuration.
+It takes one parameter:
+
+.. list-table:: Parameters to the choose_wrong_option noise type
+  :widths: 1 5 1
+  :header-rows: 1
+
+  * - Parameter
+    - Description
+    - Default
+  * - :code:`cell_probability`
+    - The probability that, for a cell in the column being configured, the wrong option is chosen.
+    - 0.01 (1%)
+
+Write the wrong ZIP code digits
+-------------------------------
+
+When reporting a ZIP code on a survey or form, people may misremember or misreport
+the digits.
+They are probably more likely to do this for the last few digits (which identify
+the small, specific area) than the first few (which will be the same over a larger area).
+
+This noise type is called :code:`write_wrong_zipcode_digits` in the configuration.
+It takes two parameters:
+
+.. list-table:: Parameters to the write_wrong_zipcode_digits noise type
+  :widths: 1 5 3
+  :header-rows: 1
+
+  * - Parameter
+    - Description
+    - Default
+  * - :code:`cell_probability`
+    - The probability of a cell being *considered* to have this noise type.
+      One way to think about this is the probability that a ZIP code is reported by someone who isn't sure of their ZIP code.
+      Whether or not there are actually any errors depends on the next parameter.
+    - 0.01 (1%)
+  * - :code:`digit_probabilities`
+    - A list of five probabilities, one for each digit in a (5-digit) ZIP code.
+      The first value in this list is the probability that the first digit of the ZIP code will be wrong
+      **given that the cell is being considered for this noise type**.
+      The second value in the list is this same probability but about the second digit, and so on.
```

Comment:
```suggestion
      The second value in the list is the corresponding probability for the second digit, and so on.
```
I think it's confusing to say it's the "same" probability. How about "corresponding"?